python/mujinwebstackclient/webstackgraphclientutils.py

```diff
@@ -93,6 +95,54 @@ def _CallSimpleGraphAPI(self, queryOrMutation, operationName, parameterNameTypeV
             log.verbose('got response from graph query: %r', data)
         return data.get(operationName)
 
+    def _CallSubscribeGraphAPI(self, operationName: str, parameterNameTypeValues: list, returnType: str, callbackFunction: Callable, fields: Optional[list[str]] = None) -> controllerwebclientraw.Subscription:
+        """
+        API for the webstack client to use for subscription.
+
+        Args:
+            operationName (string): name of the subscription operation
+            parameterNameTypeValues (list): list of tuple (parameterName, parameterType, parameterValue)
+            returnType (string): name of the return type, used to construct query fields
+            callbackFunction (func): a callback function to process the response data that is received from the subscription
+            fields (list[string]): list of fieldName to filter for
+        """
+        queryFields = ''
```

Comment:
The query construction code is almost identical to `_CallSimpleGraphAPI`. Can you extract the common parts to a separate function to avoid having repeated code?